src/core/tracing/trace_context.cpp

```
@@ -0,0 +1,142 @@
+    }
+    result.push_back('-');
+    for (const auto byte : context.spanId) {
+        appendByte(result, byte);
+    }
+    result.push_back('-');
+    appendByte(result, context.flags);
+    return result;
+}
+
+nixl::trace::TraceContext
+nixl::trace::generateTraceContext() {
```

Cppcheck (2.21.0)

[style] The function 'generateTraceContext' is never used.

(unusedFunction)